---
source: test_programs/src/lib.rs
expression: "&format!(\"success: {:?}\\nexit_code: {}\\n----- stdout -----\\n{}\\n----- stderr -----\\n{}\",\noutput.status.success(), output.status.code().unwrap_or(!0), stdout, stderr)"
---
success: false
exit_code: 1
----- stdout -----

----- stderr -----
warning: unused function foo
  ┌─ compile_failure/noirc_frontend_tests_numeric_generic_as_return_type/src/main.nr:7:8
  │
7 │     fn foo<T, let I: Field>(x: T) -> I where T: Zeroed {
  │        --- unused function
  │

error: Expected type, found numeric generic
  ┌─ compile_failure/noirc_frontend_tests_numeric_generic_as_return_type/src/main.nr:7:38
  │
7 │     fn foo<T, let I: Field>(x: T) -> I where T: Zeroed {
  │                                      - not a type
  │

Aborting due to 1 previous error
